src/dev/so_migration/snapshot_plugin_types.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as cp from 'child_process';
+
+import {
+  extractMigrationInfo,
+  getMigrationHash,
+  SavedObjectTypeMigrationInfo,
+  // TODO: how to resolve this? Where to place this script?
+  // eslint-disable-next-line @kbn/imports/no_boundary_crossing
```

Comment:
the logic is in packages are test helpers, and the dev package is not a test-related package, so the import is warning. Where should this script live?

Reply:
Our SO model serialized was put in the `test-helpers` domain because it was only meant to be used by tests at the time. With this PR, it's no longer the case, so another option would be to move/rename this package.

Doesn't seem like blocker though, the eslint ignore does the trick for now, so could easily be done as a follow-up.